system/Cookie/Cookie.php

```diff
@@ -0,0 +1,303 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Cookie;
+
+use Config\Cookie as CookieConfig;
+use Config\Services;
+
+class Cookie extends BaseCookie implements CookieInterface
+{
+	/**
+	 * Stores all cookies.
+	 *
+	 * @var array
+	 */
+	protected $cookies = [];
+
+	//--------------------------------------------------------------------
+
+	/**
+	 * Constructor.
+	 *
+	 * @param CookieConfig $config
+	 */
+	public function __construct(CookieConfig $config)
+	{
+		parent::__construct($config);
+	}
+
+	//--------------------------------------------------------------------
+
```

Comment:
This constructor can be removed if no special initializations would be done by the implementing child cookie.